embedded-graphics/src/fonts/mod.rs

```diff
@@ -134,10 +131,10 @@ where
     /// fn main() {
     ///     let mut disp = Display::default();
     ///     // Render a string with a red stroke
-    ///     let text = Font6x8::render_str("Hello world")
+    ///     let mut text = Font6x8::render_str("Hello world")
```

Comment:
Haven't checked this, but I think this `mut` could also be removed.
```suggestion
    ///     let text = Font6x8::render_str("Hello world")
```